.github/workflows/merge.yml

```
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main", "master" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  initialise:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+          
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  build_and_push:
+    needs: initialise
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Build docker image and push online
+        uses: docker/build-push-action@v4
+        with:
+          push: true
+          tags: ktunc/docker-ml-jupyterlab:latest
```

this might need to become:

```suggestion
          tags: quay.io/galaxy/docker-ml-jupyterlab:latest
```

---

And do we want to version it? So not use latest?

---

Currently, it is on Anup's account with manual versioning (latest=0.2).

Putting to Galaxy's institutional account makes sense. There is no hard-coded version anywhere in this repo, can we maybe version like this:
`docker tag quay.io/galaxy/docker-ml-jupyterlab:$(git rev-parse --short HEAD)"`

---

I added a QUAY_OAUTH_TOKEN to the secrets.

We could push on release-tags. So only if we create a tag we actually push.

---

I changed it to quay, on release, as you suggested. Lets see if it works.